src/libraries/System.IO.FileSystem.AccessControl/tests/FileSystemAclExtensionsTests.cs

```diff
@@ -380,7 +378,8 @@ private void CreateFileWithSecurity(FileInfo info, FileMode mode, FileSystemRigh
             }
```

Comment:
Do we still need to check for `PlatformDetection.IsNetFramework` or can I remove those checks?